example.env

```diff
@@ -0,0 +1,6 @@
+LFS_JWT_SECRET=qblEzhdSlDKd-jJI98SkMlrfav380D5yf5aT5Mah9A8 
+GITEA__database__NAME=gitea
```

Comment:
Maybe leave a comment like 
"# database password here", ... for "better" documentation.

Reply:
Added some documentation